core/state_transition.go

```diff
@@ -210,12 +217,46 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, failed bo
 		// error.
 		vmerr error
 	)
+	log.Debug("Applying new transaction!")
```

Comment:
Love the logging! Do you mind adding the tx to the debug output for context in case something goes wrong?

Reply:
Also if we're logging per tx (which I think is good!), it may be good to create a ticket to make sure that logging is async so as to not slow down tx processing. I assume that it is, but 🤷

Reply:
@K-Ho and I tried running txs while printing out the full execution trace every time, and it didn't seem to slow things down at first. We did end up reverting it because it seemed that it might have eventually slowed things down, but I think that's more about us printing an ungodly number of logs.